templates/master/00-master/on-prem/files/keepalived-keepalived.yaml

```diff
@@ -69,6 +69,63 @@ contents:
         fall 2
     }
 
+    # When using dual-stack with OpenStack, both IPv4 and IPv6 share the same Neutron Port,
+    # causing OVN to assume both addresses belong to the same node, which may not always be the case.
+    # To address this, we ensure that the API VIPs remain grouped through state changes,
+    # the same goes for Ingress VIPs.
+    {{- if and (eq .Infra.Status.PlatformStatus.Type "OpenStack") (or (eq .IPFamilies "DualStack") (eq .IPFamilies "DualStackIPv6Primary")) }}
```

Comment:
Let's add this comment inside the if-statement. I just looked at the keepalived.conf file for a platform that does not get the vrrp sync group and it's confusing to read this comment there.

If we put it inside if-statement, then it will be only visible for dual-stack openstack what is where we ultimately want it.

Apart from that lgtm so I will add the label as soon as it's ready

Reply:
I just fixed it.